src/pages/trade/ui/chart.tsx

```diff
@@ -7,11 +7,102 @@ import { useCandles } from '../api/candles';
 import { observer } from 'mobx-react-lite';
 import { DurationWindow, durationWindows } from '@/shared/utils/duration.ts';
 
-const ChartLoadingState = () => {
+export const ChartLoadingState = () => {
   return (
-    <div className='flex w-full h-full items-center justify-center'>
-      <div className='text-gray-500'>Loading...</div>
-    </div>
+    <svg
```

Comment:
issue: this svg does not match the svg from figma. Perhaps you can reach out to artur to extract the one in the designs.

Reply:
We can also just import svgs as react components now in the DEX:

```ts
import SVGIcon from './icon.svg';
```

Reply:
My apology - I somehow messed up both the branch name and the title - corrected now.

Reply:
The SVG in Figma is a masked version of the larger SVG - I took just the part that is visible. I'll let Arthur know if that's okay, or fix it if not.

Reply:
Visually it is the same svg.